src/openms/source/KERNEL/ConsensusMap.cpp

```diff
@@ -536,6 +571,7 @@ namespace OpenMS
            protein_identifications_ == rhs.protein_identifications_ &&
            unassigned_peptide_identifications_ == rhs.unassigned_peptide_identifications_ &&
            data_processing_ == rhs.data_processing_;
+    // @TODO: implement "operator==" for IdentificationData?
```

Comment:
yeah probably a good idea

Reply:
Not so sure. It's going to be complicated (because you can't compare two objects containing references directly - you have to dereference everything) and expensive to run on large objects. The easiest way may be to export to JSON and compare the strings.

Reply:
oh god please don't do that ;)

dereferencing / implementing operator== the key/values would be the proper way imho. not sure if high priority but something that might come up at some point

Reply:
Will save that for another time.